RecoTracker/SpecialSeedGenerators/src/OutsideInMuonSeeder.cc

```diff
@@ -41,7 +37,7 @@
 #include "TrackingTools/PatternTools/interface/TrajectoryMeasurement.h"
 #include "TrackingTools/PatternTools/interface/TrajMeasLessEstim.h"
 
-class OutsideInMuonSeeder : public edm::stream::EDProducer<> {
+class OutsideInMuonSeeder final : public edm::stream::EDProducer<> {
```

Comment:
"final" doesn't terminate inheritance from the base (upstream). All virtual methods inherited from the base still remain virtual.
It wouldn't make sense to have a new virtual method added not present in the base.